test/imports/src/core

```diff
@@ -0,0 +1 @@
+../../../src/core
```

Comment:
Hmm, why do you need these files?
`dmd` isn't used with `-conf=` and will use the import path of druntime. If you don't trust it, you can always add `-I../../import`?

Reply:
It fails: https://circleci.com/gh/dlang/druntime/1546
> Testing bug18193
> /home/circleci/dmd/generated/linux/release/64/dmd -o- -deps=generated/linux/debug/64/bug18193.done -Isrc -I../../import src/bug18193
> Error: cannot find source code for runtime library file 'object.d'
>       dmd might not be correctly installed. Run 'dmd -man' for installation instructions.
>       config file: /home/circleci/dmd/generated/linux/release/64/dmd.conf